python/lsst/ap/verify/pipeline_driver.py

```diff
@@ -257,8 +258,14 @@ def _getCollectionArguments(workspace):
         following the conventions of `sys.argv`.
     """
     butler = workspace.workButler
-    inputs = set(butler.registry.queryCollections(collectionTypes={dafButler.CollectionType.RUN}))
-    inputs.discard(workspace.runName)
+    # Hard-code the repository names because it's hard to infer the inputs from the Butler
```

Comment:
Nomenclature in Gen3 is "collection names" rather than "repository names".

Reply:
I have no idea why I said that... 😳